src/blocks/modal.css

```diff
@@ -220,7 +266,7 @@
   .modal__submit-button {
     width: 100%;
     line-height: 12.7px;
-    max-width: 627px;
+    max-width: 156px;
```

Comment:
* The **submit** button should be as `wide` as the inputs on 320px (mobile) https://snipboard.io/VsyEWL.jpg


`max-width: 627px;` should be in line 269 as it was previously